app/js/lib/foxbox/recipes.js

```diff
@@ -6,10 +6,35 @@
 const p = Object.freeze({
   // Private properties.
   settings: Symbol('settings'),
-  net: Symbol('net')
+  net: Symbol('net'),
+
+  // Recipe private properties.
+  service: Symbol('service')
 });
 
-export default class Recipe {
+export class Recipe {
```

Comment:
Just first draft of the ```Recipe``` object just to have the same "view/projection" that is used inside app views, later we can make it much smarter.